fem/libceed/ceed.hpp

```diff
@@ -181,6 +181,20 @@ struct CeedMFOperator
 void InitCeedCoeff(Coefficient* Q, Mesh &mesh, const IntegrationRule &ir,
                    CeedData* ptr);
 
+#ifdef MFEM_USE_CEED
+/// Initialize a CeedBasis and a CeedElemRestriction
+void InitCeedBasisAndRestriction(const FiniteElementSpace &fes,
+                                 const IntegrationRule &ir,
+                                 Ceed ceed, CeedBasis *basis,
+                                 CeedElemRestriction *restr);
+
+void InitCeedTensorRestriction(const FiniteElementSpace &fes,
+                               Ceed ceed, CeedElemRestriction *restr);
+#endif
+
+/// Return the path to the libCEED q-function headers.
+const std::string &GetCeedPath();
+
```

Comment:
Remove, this is already defined lower in the file.

Reply:
You can move `void InitCeedTensorRestriction(const FiniteElementSpace &fes, Ceed ceed, CeedElemRestriction *restr);`, lower in the file.

Reply:
Done.